ods_ci/tests/Resources/Common.robot

```diff
@@ -59,11 +59,15 @@ End Non JupyterLab Web Test
     Close Browser
 
 Load Json File
-    [Arguments]   ${file_path}
+    [Arguments]   ${file_path}      ${double_quotes}=${FALSE}
     ${j_file}=    Get File    ${file_path}
     ${obj}=    Evaluate    json.loads(r'''${j_file}''')    json
+    IF  ${double_quotes}
+       ${obj}=    Evaluate    json.dumps(${obj})    json
+    END
```

Comment:
2 things here:
1. what "double_quotes" have to do with the usage of json.dumps?
2. json_dumps does the exact opposite of the original intent of this keyword (i.e., loading a json into a python dictionary object). So please create another keyword for doing dumps

Reply:
So during the comparison, I'm getting an error related to differences in quotes. When converting to an object, it uses single quotes. I created a new specific keyword, but Tarun suggested adding the arguments as double quotes instead of creating a new keyword.

Reply:
the return object of dumps is a string isn't it?

Reply:
yes but it is in single quotes